packages/website/src/components/sentryCodeBlock.mdx

```diff
@@ -78,3 +80,57 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
   ```
   </TabItem>
 </Tabs>
+
+### Mobile platforms
+
+<Tabs>
+  <TabItem label="Android">
+  :::caution
+  Requires `sentry-java` version `7.4.0` or higher.
+  :::
+  ```kotlin {4}
+  SentryAndroid.init(this, options -> {
+      options.setDsn("___DSN___");
+      // You should only enable this in your development environment
+      options.isEnableSpotlight = BuildConfig.DEBUG;
+  });
+  ```
+  </TabItem>
```

Comment:
On Android you also need to allow cleartext traffic from your Emulator to Spotlight / the host machine:

`src/debug/res/xml/network.xml`

  ```xml    
<?xml version="1.0" encoding="utf-8"?>
<network-security-config>
  <domain-config cleartextTrafficPermitted="true">
    <!-- To enable spotlight we need to allow cleartext traffic from the emulator to the host machine for now-->
    <!-- See https://developer.android.com/studio/run/emulator-networking for more details -->
    <domain includeSubdomains="true">10.0.2.2</domain>
  </domain-config>
</network-security-config>
  ```

Reply:
@markushi added, ptal